packages/smart-wallet/src/smartWallet.js

```diff
@@ -304,12 +325,14 @@ export const prepareSmartWallet = (baggage, shared) => {
             offerToUsedInvitation,
             offerToPublicSubscriberPaths,
             purseBalances,
+            possiblyExitableOffers,
           } = this.state;
           currentPublishKit.publisher.publish({
             purses: [...purseBalances.values()].map(a => ({
               brand: a.brand,
               balance: a,
             })),
+            possiblyExitableOffers: mapToRecord(possiblyExitableOffers),
```

Comment:
Let's not use `mapToRecord` any more; we're trying to get rid of it.

Reply:
oh... what to use instead...
```suggestion
            possiblyExitableOffers: possiblyExitableOffers.entries(),
```

Reply:
Oops... I suppose `.entries()` returns an interator. We should materialize the list:

```suggestion
            possiblyExitableOffers: [...possiblyExitableOffers.entries()],
```